### PR TITLE
[DOCS] PhysicsDirectSpaceState

### PR DESCRIPTION
**Pull request by ghost**
I used the [Physics2DDirectSpaceState docs](http://docs.godotengine.org/en/latest/classes/class_physics2ddirectspacestate.html) as a reference, then tested each function and skimmed through source to make sure they were correct. Still, someone should check to make sure I didn't screw anything up.

### CODE REVIEW

**Comment by akien-mga** [comment]
CC @godotengine/documentation

File: doc/classes/PhysicsDirectSpaceState.xml [hunk 1]
```diff
@@ -17,6 +19,8 @@
 			<argument index="1" name="motion" type="Vector3">
 			</argument>
 			<description>
+				Check whether the shape can travel to a point. If it can, the method will return an array with two floats: The first is the distance the shape can move in that direction without colliding, and the second is the distance at which it will collide.
```
Comment (brainsick) [inline doc/classes/PhysicsDirectSpaceState.xml]: I'm still wondering about the different return values.  They both sound like the distance to the collision to me?

**Review by brainsick — COMMENTED** [review]
I have no prior game development experience so take this for what it's worth ...

File: doc/classes/PhysicsDirectSpaceState.xml (hunk 1 above)
Comment (homer666) [reply doc/classes/PhysicsDirectSpaceState.xml]: @brainsick One will be *slightly* larger than the other. How much larger depends on how often the physics engine checks for collisions, I think.

**Comment by homer666** [comment]
Oh, one thing I probably should've mentioned regarding `cast_motion()` is that the returned distances are **clamped between 0 and 1**, so if you wanted actual distances/positions within a given coordinate space you'd need to do a bit of math. Also I can't confirm this right now but if no collision is detected, the function will return `[1.0, 1.0]`.

I'll amend this commit a little later...

**Review by StraToN — APPROVED** [review]

**Pull request merged**